suite-native/transactions/src/components/TransactionDetail/TransactionDetailParametersSheet.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+
+import * as Clipboard from 'expo-clipboard';
+
+import { Blockchain, WalletAccountTransaction } from '@suite-common/wallet-types/src';
+import { Box, Card, IconButton, Text, VStack } from '@suite-native/atoms/src';
+import { Icon } from '@trezor/icons/src';
+import { getConfirmations, getFeeRate, getFeeUnits } from '@suite-common/wallet-utils';
+
+import { TransactionDetailSheet } from './TransactionDetailSheet';
+import { TransactionDetailRow } from './TransactionDetailRow';
+
+type TransactionDetailParametersSheetProps = {
+    isVisible: boolean;
+    transaction: WalletAccountTransaction;
+    handleSheetVisibility: () => void;
+    blockchain: Blockchain;
+};
+export const TransactionDetailParametersSheet = ({
+    isVisible,
+    handleSheetVisibility,
+    transaction,
+    blockchain,
+}: TransactionDetailParametersSheetProps) => {
+    const handleClickCopy = () => Clipboard.setStringAsync(transaction.txid);
+    const formattedTransactionId = `#${transaction.txid.slice(0, 11)}...`;
```

Comment:
Did you consider Text prop `ellipsizeMode`? I would avoid doing this and just set this prop together with width if there is no bottlenecks with it. 

Anyway, this will be handled by formatter soon-ish

Reply:
If you use ellipsizeMode, then you are not in control of how many symbols are displayed. I used this basic slice to exactly specify the length. Also, there is a `#` prefix anyway, so some kind of formating would be necessary anyway.

Reply:
I think maybe `ellipsizeMode` will be really better, I think whole point of truncating this is because it's long and there is not enough space, but there are people with huge phones and we also support tables "somehow" so there is no reason to truncate it for them.

Reply:
Exactly what @Nodonisko says. Plus there is no reason to have 11 characters specifically. If you want to use function, please create exportable constant that is used for truncating transaction ids globally. But I would just go with the ellipsis mode preferrably